content/05kapitel.tex

```diff
@@ -123,6 +123,108 @@ \subsubsection{Übersicht der Master \ac{CI}-Pipline}
 
 \section{Mockups}
 
+\subsection{Aktivitätsdiagramme}
```

Comment:
Im Ticket war spezifiziert, dass das in die Mockups seite von Theo kommt (haben wir eigentlich auch noch extra letzten Donnerstag besprochen...)

Reply:
Removing duplications, should be fine otherwise